### ISSUE TITLE
Survey Flow Update - Make the prompts consistent

### ISSUE THREAD

**Issue by JayceKongg** [AppHatchery/Fabla-Front-end#137]
# Problem
Tips such as "use the slider to rate" appear in the second and third questions. However, the first question also needs participants to use the slider. 

For example, the first question is "How do you FEEL PHYSICALLY right now? " The second question is "How do you FEEL EMOTIONALLY right now? Please use the slider to rate how PLEASANT or UNPLEASANT you feel emotionally." And the third question is "Please use the slider to rate the INTENSITY of the overall emotion you are experiencing right now"

We could remove these guidelines as they apparently have to use sliders. Or have the guidelines only appear in the first question.

# Font Check (Size, weight) #127 
Tag the similar issue here

# Suggested Question List
**Note: According to IRB, we need to remove the alcohol question from the list.**

1. How do you FEEL PHYSICALLY right now? 
2. How do you FEEL EMOTIONALLY right now?
3. Please rate the INTENSITY of the overall emotion you are experiencing right now.
4. I felt lonely today.
5. I felt left out today.
6. How much social interaction did you have today?
7. I felt understood/cared for by others today.
8. I felt stressed today.
9. Which of the following best describes where you are physically right now? (Check all that apply)
10. How many people are around you right not?
11. Talk about your day today.
12. Talk about today’s most stressful experience, no matter how small.

**Comment by yago1994** [comment]
I agree it is pretty self-explanatory, however to make sure it is clear we do have to include them. I'd go with your proposal of only including it in the first question. 
How do you FEEL PHYSICALLY right now? (Please use the slider to rate how BAD or GOOD you feel physically).

**Comment by JayceKongg** [comment]
@Yago How about we make a universal guide that can be applicable to all questions using sliders (i.e., Please use the slider to rate)?  I've modified the information hierarchy on the questions page so that we can put the guide below the questions. (Vertical/Horizontal slider TBD)

<img width="600" alt="Screenshot 2023-10-16 at 16 48 45" src="https://github.com/AppHatchery/Audio-Diaries-Flutter/assets/134336047/99b980a0-f875-4e34-a695-325b522773dd">

**Comment by yago1994** [comment]
Looks good to me but it does add more customization, let's keep it how we have it for now to simplify it for the developers. There are a lot of changes going on right now.

I'm happy to include it after release, if we are going to have fixed questions (i.e. multiple choice, single choice, sliders, etc) it should be pretty standard to include a small blurb that matches the question class automatically.

**Comment by JayceKongg** [comment]
Agree, let's do this. We include the guideline "Please use the slider to rate how BAD or GOOD you feel physically" in the first question. I have 2 proposals for that.

1) Have a separate line to display the guideline, the font size and color can be found in #127 
<img width="300" alt="Screenshot 2023-10-17 at 11 23 07" src="https://github.com/AppHatchery/Audio-Diaries-Flutter/assets/134336047/15a1c2c6-cb26-4ac1-b7b3-5fd89aa4e2b5">

2) If 1) needs too much customization, let's put the guideline after the title. (The same that we use now)
<img width="300" alt="Screenshot 2023-10-17 at 11 25 23" src="https://github.com/AppHatchery/Audio-Diaries-Flutter/assets/134336047/e3703b4c-1b53-46d0-a6c3-238543d445ec">

**Comment by yago1994** [comment]
@JayceKongg The first option is better visually speaking, but the second one is a faster change (i.e. would take 5 mins). I suggest we go with the second one for now and see where we are on development at the end of the week.

**Comment by Wiza-Munthali** [comment]
Closing - outdated
